projects/go-tester/src/app/components/test-page-1/test-page-1.component.ts

```diff
@@ -40,18 +39,16 @@ export class TestPage1Component implements OnInit {
         totalCount: data.totalCount,
         sortable: false
       });
-      this.tableLoading = false;
-    });
 
-    this.appService.getMockData(new GoTableConfig()).subscribe((data: any) => {
```

Comment:
I removed a lot of this stuff just to clean it up a bit, we didn't really need to make this call twice.